docs/package-hub/package-builder/gem-builder.md

```diff
@@ -476,6 +477,10 @@ You can go ahead and preview the component to see how it looks. Note some Gem ex
 
 That’s all the code for our example Transformation, the Limit Gem. We walked through the package and import statements, parent class, and properties class. We explored the required methods dialog, validation, onChange, (de)serializeProperty. Finally we saw the Limit Gem’s component code. Now we have a basic understanding of the components needed for any Transformation Gem.
 
+:::info
+If you are using an existing Gem as a guide to creating your new Gem, you will need to change the following at a minimum: ComponentSpec, ComponentProperties, ComponentCode.
```

Comment:
Suggest using unordered list for the three items.

Reply:
I tried using bullets within the info callout, but our whitespace checker changed to this:
![image](https://github.com/user-attachments/assets/f470b270-9bf6-42fe-884d-2154c02e42a7)